plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/app/Application.js

```diff
@@ -238,8 +238,8 @@ Ext.define('NX.app.Application', {
 
     remotingProvider = Ext.direct.Manager.addProvider(NX.direct.api.REMOTING_API);
 
-    // disable buffering of method calls
-    remotingProvider.enableBuffer = false;
+    // configure Ext.Direct buffer-window milliseconds
+    remotingProvider.enableBuffer = 100;
```

Comment:
Is this going to have any conflict with existing code? It was set to false as part of browse-ui, although I'm not sure why it was necessary: https://github.com/sonatype/nexus-oss/commit/ed97ec82bb17190b36ad45aed67604a485628a78